### ISSUE TITLE
Getting 404 on endpoints when using the docker image V2.0.2

### ISSUE THREAD

**Issue by twittyc** [idealista/prom2teams#61]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Checked that your issue isn't already filled: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aidealista
    * Checked that there is not already provided the described functionality

### Description

Getting 404 on endpoints when using the docker image V2.0.2

### Steps to Reproduce

1. Navigate to http://host:port/v2

Hey guys, the documentation is a little confusing on what the appropriate endpoint is for this application. When using the docker image, I am unable to hit the endpoints. I'm getting a 404. I tried http://host:port/v2 and http://host:port/v2/Connector. Also I wasn't able to hit the swagger-ui. I was hoping for some clarification on this. Thanks!

**Comment by jnogol** [comment]
Yep, we've found an odd behaviour, and the host you send the requests to needs to be the same as the `Host` field specified in the configuration file. So, in the Docker case, you need to set `PROM2TEAMS_HOST` as the container IP. For example:

``` bash
$ docker run -it -d --name prom2teams -e PROM2TEAMS_CONNECTOR="myConnectorURL" -e PROM2TEAMS_HOST="172.17.0.2" -p 8089:8089 idealista/prom2teams:2.0.2
```

Now you could navigate to http://172.17.0.2:8089/ and you'll see the swagger interface. Sending alarms to the endpoints will work as expected.

We will take a look and try to improve this behaviour as soon as possible. Feel free to contribute sending us a PR.

Regards,
Jose

**Comment by twittyc** [comment]
I'm trying to run this on Kubernetes. So ideally the app doesn't care what the hostname is if it's running in a container. As Kubernetes dynamically assigns IP addresses to containers when they are created and uses service manifests to allow the containers to communicate to each other using a DNS name. I can try to take a look.

**Comment by a-zen** [comment]
Ah this explains it. I was wondering where this odd behaivour came from.

**Comment by evidyasagar** [comment]
Prerequisite:
tried deploying the docker container using latest one, however, i am getting 405 on endpoints when using the docker image V2.0.3. and teams should get integrated with alert manager

Steps to reproduce:
Tried deploying latest prom2teams tool of 2.0.3 version on one of the vm and returning with 405 error. 
2021-04-12 13:48:50,852 - werkzeug - INFO - 172.17.0.1 - - [12/Apr/2021 13:48:50] "GET /v2/Connector HTTP/1.1" 405 -
I get this when i try to curl the prom2teams link 
curl http://localhost:8089/v2/Connector.
{"message": "The method is not allowed for the requested URL."}

Also, I tried but  I wasn't able to hit the swagger-ui.  Would you mind providing some solution around this new version if there's a fix already in place. Thanks!
